### DISCUSSION TITLE
Passwords + 2FA

### DISCUSSION THREAD

**Discussion by ethndotsh** [spacedriveapp/spacedrive#262] [Feature Request]
Able to store and use passwords and TOTP tokens in Spacedrive. Maybe even integrate with a Chrome extension?

This would be great as it would reduce the number of apps I use by ~6 (file storage, passwords, 2fa)

**Comment by oscartbeaumont** [comment]
It would be cool but I am not sure it will ever happen. If you have a single service that does everything, it most likely can't do everything well because there is only so much engineering power you can put behind a single application.

Although who knows maybe we will one day, that's not my decision.

**Reply by Crushoverride007** [reply]
or just automating a manual task if its possible
